### PR TITLE
Feature: Page SelectOption default selection

### PR DESCRIPTION
**Pull request by yob-yob**
This PR will allow setting the default selected option for the `SelectAction` under Page Actions.

I had a problem where when I refresh the page, the default option I wanted it to show is not selected, making it confusing to my users on what is the selected default option, because my table is showing a different data.

### CODE REVIEW

File: packages/admin/resources/views/pages/actions/select-action.blade.php [hunk 1]
```diff
@@ -16,9 +16,15 @@
         @endif
 
         @foreach ($getOptions() as $value => $label)
-            <option value="{{ $value }}">
-                {{ $label }}
-            </option>
+            @if(($selected = $getSelected()) !== null)
+              @if($selected == $value)
```
Comment (Saifallak) [inline packages/admin/resources/views/pages/actions/select-action.blade.php:20]: Maybe use @selected blade directive ?

Comment (danharrin) [reply packages/admin/resources/views/pages/actions/select-action.blade.php:20]: Is that supported in Laravel 8?

Comment (Saifallak) [reply packages/admin/resources/views/pages/actions/select-action.blade.php:20]: Ops, nope, it was added 22 feb 2022, so L9.
Maybe for filament v3 then

**Comment by danharrin** [comment]
Hey, you should actually accomplish this by setting the Livewire public property that this action is bound to inside `mount()`. This action is actually for internal use, which is why it's not been documented, but that's how we use it 👍 Setting a `selected` attribute like this could conflict with the `wire:model`

**Comment by yob-yob** [comment]
Oh My God... I'm honestly sorry, I did not noticed the `wire:model`... Big Mistake on part...

Sorry...

**Pull request closed without merging**